checker/jtreg/index/UnneededSuppressionsTest.java

```diff
@@ -36,4 +36,12 @@ void method5() {}
 
     @SuppressWarnings("purity")
     void method6() {}
+
+    // TODO: It is a bug that this is not reported as an unused suppression
+    @SuppressWarnings("index:foo.bar.baz")
+    void method7() {}
+
+    // TODO: It is a bug that this is not reported as an unused suppression
+    @SuppressWarnings("allcheckers:purity.not.deterministic.call")
```

Comment:
A checker besides the Index Checker could use this warning suppression, so it isn't reported.

Reply:
1. Users get no warning about unused warning suppressions with the allcheckers: prefix, which is not documented.
2. Most users run just a few checkers, so they can suppress the warning for each relevant checker.

Do you think the current undocumented behavior is better than the behavior proposed in this TODO?  (Your comments don't indicate your position on this.)

Reply:
I understood the TODO as a question of whether this is a bug or desired behavior.  My point was that it is not a bug that there was no warning on this suppression.  We designed this option to only issue warnings when the suppression was clearly not needed, so this is the desired behavior.